package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "start": "node src/app.js",
     "start:dev": "nodemon src/app.js",
```

Comment:
This script. I don't understand how to start it. 
Is it so different from just using **"dev"**

Reply:
I think it is the same thing